.github/workflows/pyre.yml

```diff
@@ -14,31 +14,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
 
-      - name: Install Dependencies
+      - name: Install Dependencies and pyre-check
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install -r requirements.txt
-          python -m pip install cython
-          python -m pip install flask flask_cors graphql-core typing_inspect
+          pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install cython flask flask_cors graphql-core typing_inspect
           VERSION=$(grep "version" .pyre_configuration | sed -n -e 's/.*\(0\.0\.[0-9]*\).*/\1/p')
-          python -m pip install pyre-check-nightly==$VERSION
+          pip install pyre-check-nightly==$VERSION
```

Comment:
Those formatting changes should probably have been in a different pull request. It doesn't seem related to your changes and just hinders code review.

Reply:
Thanks. I will keep this in mind :)

Reply:
@arthaud I added this as an additional commit for now, but if you want - I can move it to a new branch, wait the first commit to be merged, and then introduce the other two commits in separate PRs.